app/Models/User.php

```diff
@@ -280,7 +280,6 @@ public function teams(): BelongsToMany
         return $this
             ->belongsToMany(Team::class, 'team_user', 'user_id', 'team_id')
             ->using(TeamUser::class)
-            ->limit(1)
```

Comment:
Why was this removed? This was intentionally placed here for now as we only want Users to be related to a single team.

Reply:
Explained in office hours. Good to go.